lib/content-services/social/services/rating.service.spec.ts

```diff
@@ -24,17 +24,8 @@ describe('Rating service', () => {
 
     let service;
 
-    beforeEach(async(() => {
-        TestBed.configureTestingModule({
-            imports: [],
-            providers: [
-                RatingService
-            ]
-        }).compileComponents();
-    }));
-
     beforeEach(() => {
-        service = TestBed.get(RatingService);
+        service = new RatingService(new AlfrescoApiService(new AppConfigService(null), new StorageService()));
```

Comment:
I think you can use TestBed here, but without calling ".compileComponents()", should be fast.